app/components/transcript/program/program.html

```diff
@@ -1,4 +1,4 @@
 <div class="program">
-  <div class="level">Master of Science</div> <!-- todo: parameterize this! -->
+  <div class="level">Master of Business Administration</div> <!-- todo: parameterize this! -->
```

Comment:
This should use `{{program.degree_level}}`, similar to the program title below. Can you update this to draw that value, and then update the JSON to be "Master of Business Administration" instead of "Bachelor of Professional Studies"?